storage/sql/migrations/updates/20190204170311_add_asset_field_as_json_to_trs_table.sql

```diff
@@ -12,4 +12,13 @@
  * Removal or modification of this copyright notice is prohibited.
  */
 
-INSERT INTO "transfer" ("transactionId", "data") VALUES ${values:raw};
+
+/*
+  DESCRIPTION: Add asset Column to trs.
+
+  PARAMETERS: None
+*/
+
+ALTER TABLE "trs" ADD COLUMN IF NOT EXISTS "asset" jsonb;
+
+CREATE INDEX IF NOT EXISTS "trs_asset" ON "trs" USING gin ("asset");
```

Comment:
`GIN` index is used explicitly for full text search. Are we going to use full-text search in `asset`? If not then I believe we can still use `btree`.

Reply:
I used `GIN` as per postgres documentation: `GIN indexes can be used to efficiently search for keys or key/value pairs occurring within a large number of jsonb documents`

REF.: https://www.postgresql.org/docs/10/datatype-json.html